Client/Ecosia/UI/MyImpact/MultiplyImpact.swift

```diff
@@ -232,8 +232,19 @@ final class MultiplyImpact: UIViewController, Themeable {
         dismiss(animated: true)
     }
     
-    @objc private func inviteFriends() {
-        Analytics.shared.sendInvite()
+    @objc private func inviteFriends(_ button: UIButton) {
+        #warning("Invite link not defined yet")
+        let inviteLink = Environment.current.referralClaims
+        
+        let share = UIActivityViewController(activityItems: [inviteLink], applicationActivities: nil)
```

Comment:
I am wondering - shouldn't we also hand over some localised text message?

> Hey there, want to plant trees with me? If you get the app, each of us will contribute to planting extra trees with Ecosia!
> (1) Download Ecosia on your iPhone for free
> (2) Click my invitation link so we plant extra trees each: Open invitation link
> Let's do this 😊

I think only the link won't be enough 🤔

Reply:
I agree, the problem is that it is no defined yet, at least not in the ticket, but I can add that one and we update accordingly 👍